test_pull_requests

```diff
@@ -458,6 +460,22 @@ module Hub
       comment += extended_tests if extended_tests
       recreate_comment(pull_id, bot_comment['id'], repo, comment)
 
+      bot_sha_comment = get_comment_with_prefix(pull_id, repo, BASE_SHA_PREFIX)
```

Comment:
That occurred to me; I'm trying to figure out a decent way to add it to the `SUCCESS` comment so we can parse it out here without breaking pull requests in progress: https://github.com/dobbymoodge/test-pull-requests/blob/50e86f0cb8218173c40d9b91c9945249e11764e6/test_pull_requests#L1230

I was thinking of appending "` base_ref:<sha-commit-id>`" to the comment as part of `--mark_test_success`, and then writing that line like:
`if comment =~ /^#{settings['test_prefix']} SUCCESS \(([^\)]+)\) ?(base_ref:)?([a-f0-9]{40})?/`
and then checking if `$3` is empty. On the other hand, I could leave the regex as-is and then below check:
`if comment =~ /base_ref:([a-f0-9]{40})`

Which do you like?